### DISCUSSION TITLE
Option to disable one way roads

### DISCUSSION THREAD

**Discussion by Shituation** [OpenTTD/OpenTTD#10186] [Features]
Because they don't have a real use in the game and are there only to be abused by trolls, by sending your trucks into railroads to be destroyed.

**Comment by 2TallTyler** [comment]
Trolls can still destroy your vehicles without one-way roads, or do any number of things with terraforming, buying land, etc. I think a better solution is not playing with trolls.

**Reply by Shituation** [reply]
https://github.com/OpenTTD/OpenTTD/discussions/10187.

What a great comment son. Really thoughtful and helpful.
I'm a public server hoster btw.

**Reply by andythenorth** [reply]
By hosting public servers, you have chosen to play an unwinnable game.

Military maxim: don't reinforce failure.  Don't put resources into a battle that is already lost and over.

Good luck out there 😄

**Comment by LC-Zorg** [comment]
Just because you don't see a use for one-way roads doesn't mean no one else sees it. ;) What's more, the construction of one-way roads is one of the ways to protect yourself from building tracks on the road and destroying vehicles by trolls.

However, I understand the problem. More than once I saw whole groups of vehicles blocked on city roads. 

I would like to propose another solution to set a minimum rating in the city for the construction of a one-way road. For example, +250, which is slightly more than after planting trees. Thanks to this, a one-way road in the city could only be built by a player who has been operating in the city for some time, but not the one who just wants to direct traffic... to the black hole.
